diff -crB include/application.config.php include.cp/application.config.php
*** include/application.config.php	2024-01-24 12:00:00.000000000 +0700
--- include.cp/application.config.php	2024-05-20 20:44:56.180951202 +0700
***************
*** 22,28 ****
   **[N]**/ ?>
  <?
  /* Awal tahun pendataan menggunakan SISFO JIBAS */
! $G_START_YEAR='2011';
  
  /* Alamat Server aplikasi SISFO JIBAS 
       Alamat ini digunakan untuk menampilkan header cetak di laporan-laporan yang disediakan SISFO JIBAS 
--- 22,28 ----
   **[N]**/ ?>
  <?
  /* Awal tahun pendataan menggunakan SISFO JIBAS */
! $G_START_YEAR=getenv("G_START_YEAR");
  
  /* Alamat Server aplikasi SISFO JIBAS 
       Alamat ini digunakan untuk menampilkan header cetak di laporan-laporan yang disediakan SISFO JIBAS 
***************
*** 32,39 ****
  $G_SERVER_ADDR='localhost/jibas';
  
  /* Sistem operasi yang digunakan ( win | lin ) */
! $G_OS='win';
  
  /* Lokasi Sekolah */
  $G_LOKASI="Bandung";
! ?>
\ No newline at end of file
--- 32,39 ----
  $G_SERVER_ADDR='localhost/jibas';
  
  /* Sistem operasi yang digunakan ( win | lin ) */
! $G_OS='lin';
  
  /* Lokasi Sekolah */
  $G_LOKASI="Bandung";
! ?>
diff -crB include/database.config.php include.cp/database.config.php
*** include/database.config.php	2024-01-24 12:00:00.000000000 +0700
--- include.cp/database.config.php	2024-05-20 20:39:56.122858970 +0700
***************
*** 23,37 ****
  <?
  
  /* Alamat basis data MySQL JIBAS */
! $db_host='localhost:3434';
  
  /* User basis data MySQL JIBAS */
  $db_user='root';
  
  /* Password basis data MySQL JIBAS */
! $db_pass='kebersamaan';
  
  /* Basis data default JIBAS */
! $db_name='jbsakad';
  
! ?>
\ No newline at end of file
--- 23,37 ----
  <?
  
  /* Alamat basis data MySQL JIBAS */
! $db_host='database';
  
  /* User basis data MySQL JIBAS */
  $db_user='root';
  
  /* Password basis data MySQL JIBAS */
! $db_pass=getenv("MYSQL_ROOT_PASSWORD");
  
  /* Basis data default JIBAS */
! $db_name=getenv("MYSQL_DATABASE");
  
! ?>
diff -crB include/filesharing.config.php include.cp/filesharing.config.php
*** include/filesharing.config.php	2024-01-24 12:00:00.000000000 +0700
--- include.cp/filesharing.config.php	2024-05-20 20:51:31.942821701 +0700
***************
*** 24,34 ****
  /* Direktori tempat penyimpanan file-file FileSharing
     Direktori slashes di sistem operasi Linux menggunakan /
     sedangkan di Windows bisa menggunakan \\ dan /   */
! $FILESHARE_UPLOAD_DIR = "C:\\YIM\\JIBAS\\xampp\\htdocs\\filesharing\\";
   
  /* Alamat web based dari direktori tempat penyimpanan file-file FileSharing 
     PERHATIAN:
       UNTUK PENGGUNAAN MULTIUSER di LOCAL AREA NETWORK
       JANGAN MENGGUNAKAN localhost TETAPI GUNAKAN IP ADDRESS atau HOSTNAME */
! $FILESHARE_ADDR = "http://localhost/filesharing/";
! ?>
\ No newline at end of file
--- 24,34 ----
  /* Direktori tempat penyimpanan file-file FileSharing
     Direktori slashes di sistem operasi Linux menggunakan /
     sedangkan di Windows bisa menggunakan \\ dan /   */
! $FILESHARE_UPLOAD_DIR = "/var/www/html/jibas/filesharing";
   
  /* Alamat web based dari direktori tempat penyimpanan file-file FileSharing 
     PERHATIAN:
       UNTUK PENGGUNAAN MULTIUSER di LOCAL AREA NETWORK
       JANGAN MENGGUNAKAN localhost TETAPI GUNAKAN IP ADDRESS atau HOSTNAME */
! $FILESHARE_ADDR = "http://localhost/jibas/filesharing/";
! ?>
diff -crB include/school.config.php include.cp/school.config.php
*** include/school.config.php	2024-05-20 20:58:20.501413592 +0700
--- include.cp/school.config.php	2024-05-20 20:58:05.608010135 +0700
***************
*** 27,35 ****
     Simpan Gambar Logo di direktori images
     Jika tidak menggunakan logo, masukkan nama gambar nologo.png
  */
! $G_LOGO_DEPAN_KIRI="tutwuri.png"; // nama file gambarnya, jika tidak ada gunakan nama file nologo.png
! $G_LOGO_DEPAN_KANAN="tutwuri.png";
! $G_JUDUL_DEPAN_1="SEKOLAH PENDIDIKAN INDONESIA";
! $G_JUDUL_DEPAN_2="YAYASAN PENDIDIKAN INDONESIA";
! $G_JUDUL_DEPAN_3="(atur kembali logo dan identitas sekolah di file school.config.php <br> untuk lebih jelasnya silahkan baca file readme.txt)";
  ?>
--- 27,35 ----
     Simpan Gambar Logo di direktori images
     Jika tidak menggunakan logo, masukkan nama gambar nologo.png
  */
! $G_LOGO_DEPAN_KIRI=getenv("G_LOGO_DEPAN_KIRI"); // nama file gambarnya, jika tidak ada gunakan nama file nologo.png
! $G_LOGO_DEPAN_KANAN=getenv("G_LOGO_DEPAN_KANAN");
! $G_JUDUL_DEPAN_1=getenv("G_JUDUL_DEPAN_1");
! $G_JUDUL_DEPAN_2=getenv("G_JUDUL_DEPAN_2");
! $G_JUDUL_DEPAN_3=getenv("G_JUDUL_DEPAN_3");
  ?>
diff -crB include/system.config.php include.cp/system.config.php
*** include/system.config.php	2024-05-20 21:01:00.832168291 +0700
--- include.cp/system.config.php	2024-05-20 21:00:51.332123582 +0700
***************
*** 27,33 ****
  	Untuk daerah Indonesa bagian tengah (WITA) gunakan Asia/Makassar
  	Untuk daerah Indonesa bagian timur (WIT) gunakan Asia/Jayapura
  */
! define('JIBAS_TIMEZONE','Asia/Jakarta');
  
  date_default_timezone_set(JIBAS_TIMEZONE);
  ?>
--- 27,33 ----
  	Untuk daerah Indonesa bagian tengah (WITA) gunakan Asia/Makassar
  	Untuk daerah Indonesa bagian timur (WIT) gunakan Asia/Jayapura
  */
! define('JIBAS_TIMEZONE', getenv("JIBAS_TIMEZONE"));
  
  date_default_timezone_set(JIBAS_TIMEZONE);
  ?>
